flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/FlinkDynamicTableFactory.java

```diff
@@ -126,7 +133,9 @@ public DynamicTableSink createDynamicTableSink(Context context) {
           objectPath.getObjectName());
     }
 
-    return new IcebergTableSink(tableLoader, tableSchema);
+    boolean upsert = PropertyUtil.propertyAsBoolean(tableProps, WRITE_UPSERT_ENABLED.key(), false);
```

Comment:
Can we add a constant for the default value here, similar to `TableProperties.UPSERT_ENABLED_DEFAULT`?

Reply:
Will do.